x-pack/plugins/security_solution/server/endpoint/lib/policy/saved_object_mappings.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectsType } from '../../../../../../../src/core/server/types';
+import { PACKAGE_POLICY_SAVED_OBJECT_TYPE } from '../../../../../ingest_manager/common';
+import { migratePackagePolicyToV7110 } from './to_v7_11.0';
+
+export const packagePolicyType: SavedObjectsType = {
```

Comment:
is it a problem that we define this here as well as in Ingest?  If it is OK, should we import the mapping, clone it, and add our own migration?  What happens if there are 2 migrations for 7.11?  One in Ingest and one in Security?

cc @jfsiii @jen-huang

Reply:
judging from the test failures, it certainly looks like this is an issue

https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/82038/execution/node/636/log/

Reply:
hmmm so then should i just have the code for the migration in endpoint, but then add our migration line to the one in ingest for package policy saved objects then?

Reply:
> hmmm so then should i just have the code for the migration in endpoint, but then add our migration line to the one in ingest for package policy saved objects then?

This seems like what needs to happen.  We can't define a duplicate mapping for an existing SO.  Unless there's a way that we import the SO mapping from ingest in our app and simply add another migration to it.